etc/payment.xml

```diff
@@ -7,5 +7,8 @@
         <method name="braintree_googlepay">
             <allow_multiple_address>0</allow_multiple_address>
         </method>
+        <method name="braintree_venmo">
+            <allow_multiple_address>0</allow_multiple_address>
+        </method>
     </methods>
 </payment>
```

Comment:
Does MCS specify if we should be ending .xml files with a `\n`? I've been doing it (assuming I remember) but is it something we should start doing as habit?

Reply:
I've always done it as habit with PHP files, and Github always picks up on it for other file types. Can't hurt to just do it everywhere for consistency?